sccp/figures/figureS2.py

```diff
@@ -30,21 +30,24 @@ def makeFigure():
     subplotLabel(ax)
 
     # Import of data
-    data, cell_types, _ = load_lupus_data()  # don't need to get patient color mappings
-    rank = 30
-    cellState = 28
-    cmp = 28
+    data, obs = load_lupus_data(every_n=10)  # don't need to get patient color mappings
+    rank = 5
+    cellState = 4
```

Comment:
You won't need cell state anymore so you should remove them from your functions

Reply:
@andrewram4287 looking back through this and thinking maybe I should just keep the cell state version in here at least until your PR about proj x B gets merged to main (in which case I could just merge main)-- could also include those changes in this but since that is already a separate PR that might be easier